irony-completion.el

```diff
@@ -252,11 +268,12 @@ A candidate is composed of the following elements:
  2. The [result-]type of the candidate, if any.
  3. If non-nil, contains the Doxygen brief documentation of the
     candidate.
- 4. The signature of the candidate excluding the result-type
+ 4. The accessiblity of the candidate.
```

Comment:
i have attempted to do this. i am not sure about the elisp implementation at all, but it seems to work for me so far.

the only issue i have with this is that now we are kind of having n^2 since we loop through the candidates in irony-completion-candidates and in the company backend.. but perhaps this is negligible given the size of the data to process...